docs/docs/schemas/introduction.md

```diff
@@ -125,6 +96,56 @@ end
 The `#data` method of an HTTP request object returns a hash-like object containing the request data: this object is automatically initialized from any form data or JSON data contained in the request body.
 :::
 
+### Rendering schema errors within forms
```

Comment:
Shouldn't this section be named `Rendering schemas as forms` since the intent is to showcase that schemas can indeed be used in templates in order to display the corresponding forms?